tests/functional/encryption/test_encryption_keyrotation.py

```diff
@@ -30,7 +35,6 @@
 @encryption_at_rest_required
 @skipif_kms_deployment
```

Comment:
Should be moved on function level where it's needed to not skip new test if it needs to run on KMS